contracts/EscrowAndStaking/StakingThales.sol

```diff
@@ -128,6 +129,12 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
     IStakingThalesBonusRewardsManager public stakingThalesBonusRewardsManager;
     IParlayAMMLiquidityPool public parlayAMMLiquidityPool;
 
+    bool public readOnlyMode;
+    bool public closingPeriodInProgress;
+    uint public closingPeriodPauseTime;
+    address public ccipCollector;
+    address public safeBoxBuffer;
```

Comment:
add to addressMAnager and use it from there. Make this variable private.

Reply:
added